src/App.css

```diff
@@ -1,23 +1,223 @@
-.App {
-  text-align: center;
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+body {
+  font-family: 'Montserrat', sans-serif;
+  background-color: #fafafa;
 }
 
-.App-header {
-  background-color: #282c34;
+/* Header styles */
+
+header {
+  background-color: white;
+  border-bottom: 1px solid #e8e8e8;
   display: flex;
-  flex-direction: column;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  justify-content: space-between;
+  margin-bottom: 2.375rem;
 }
 
-nav {
+.title-nav {
+  width: 45vw;
   display: flex;
+  align-items: center;
+  margin: 0 0 0 7vw;
+}
+
+.title {
+  white-space: nowrap;
+  color: #0290ff;
+  font-size: 1.8rem;
+  width: 15rem;
+  font-weight: bold;
+  padding: 1rem 0;
+}
+
+nav {
+  margin-left: 0.7rem;
+}
+
+.login {
+  justify-content: flex-end;
+  margin-right: 7vw;
 }
 
 .link {
-  color: #61dafb;
   text-decoration: none;
+  text-transform: uppercase;
+  font-size: 0.83rem;
+  color: #121212;
+  margin-left: 1rem;
   padding: 1rem;
 }
+
+/* Book styles */
+
+.book {
+  height: 6.625rem;
+  background-color: white;
+  border: 1px solid #e8e8e8;
+  display: grid;
+  grid-template-columns: 4fr 3fr 3fr;
+  margin: 0 7vw 0.938rem 7vw;
+  padding: 2rem;
+}
+
+.category {
+  color: #121212;
+  font-size: 0.875rem;
+  opacity: 0.5;
+  margin: 0 0 0.25rem 0;
+  font-weight: bold;
+}
+
+.name {
+  font-family: 'Roboto Slab', sans-serif;
+  font-weight: bold;
+  color: #121212;
+  font-size: 1.375rem;
+  margin: 0;
+}
+
+.author {
+  font-family: 'Roboto Slab', sans-serif;
+  font-weight: bold;
+  color: #4386bf;
+  font-size: 0.875rem;
+  margin: 0;
+}
+
+.left button {
+  height: 1.188rem;
```

Comment:
- [ ] Please remove the height because in this case is not doing anything. To make your code clean it is better to remove unnecessary code. Keep coding 💪